app/src/main/java/com/pajato/android/gamechat/common/InvitationManager.java

```diff
@@ -143,8 +143,9 @@ public void onAuthenticationChange(final AuthenticationChangeEvent event) {
 
     /** Handle the room profile change */
     @Subscribe public void onRoomProfileChange(@NonNull final ProfileRoomChangeEvent event) {
-        // If this room is one we care about, add the current account to the room memberIdList.
+        // If this room is in the mInvitedGroups list, add the current account to the room memberIdList.
```

Comment:
fwiw, I try to avoid using literal variable, method, and class names in comments in the hopes that when I refactor the names the comments do not need to be modified. The context usually provide clarity.